src/lib/viewers/media/MediaBaseViewer.js

```diff
@@ -703,6 +780,7 @@ class MediaBaseViewer extends BaseViewer {
         // Make sure it's within bounds
         newTime = Math.max(0, Math.min(newTime, this.mediaEl.duration));
         this.setMediaTime(newTime);
+        this.metrics[MEDIA_METRIC.seeked] = true;
```

Comment:
Can we centralize our handling of this to the `seeked` handler?

Reply:
Seems reasonable. I'll double check that the `seekHandler` gets called during `quickSeek`